### PR TITLE
Closes #2622: Extend OpenAPI with History and Routing Doc

### LINKED ISSUE #2622: Add History and Routing Doc to OpenAPI Documentation

### PR DESCRIPTION
**Pull request by jamesrdi**
---
Release Notes:

```

```
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → (Closes) #&lt;Issue Number&gt;: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: rest/taskana-rest-spring-example-boot/pom.xml [hunk 1]
```diff
@@ -55,6 +55,21 @@
       <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
       <version>${version.spring-openapi}</version>
     </dependency>
+    <dependency>
```
Comment (holgerhagen) [inline rest/taskana-rest-spring-example-boot/pom.xml:58]: Aren't those dependencies for local testing only? If they are, please remove them.

**Review by holgerhagen — COMMENTED** [review]
Didn't you say that you have removed the Rest Docs stuff? I still see the dependencies. Or is this another PR?

File: rest/taskana-rest-spring-example-boot/pom.xml (hunk 1 above)
Comment (jamesrdi) [reply rest/taskana-rest-spring-example-boot/pom.xml:58]: Yes you are right, I will remove them

**Comment by jamesrdi** [comment]
> Didn't you say that you have removed the Rest Docs stuff? I still see the dependencies. Or is this another PR?

It will be in another PR

**Review by holgerhagen — APPROVED** [review]

**Pull request closed without merging**